Libraries/Lists/VirtualizedList.js

```diff
@@ -349,6 +352,19 @@ function windowSizeOrDefault(windowSize: ?number) {
   return windowSize ?? 21;
 }
 
+function findLastWhere<T>(
+  arr: Array<T>,
+  pred: (element: T) => boolean,
+): T | null {
+  for (let i = arr.length - 1; i >= 0; i--) {
+    if (pred(arr[i])) {
+      return arr[i];
+    }
+  }
+
+  return null;
+}
+
```

Comment:
Will move this somewhere more shared if it stays after fixes to "no getLayoutView clamping"